libmscore/rest.h

```diff
@@ -30,14 +30,16 @@ class Rest : public ChordRest {
       // values calculated by layout:
       SymId _sym;
       int dotline    { -1  };       // depends on rest symbol
-      qreal _mmWidth;               // width of multi measure rest
+      qreal _mmWidth;               // width of multimeasure rest
+      qreal _yPosMMRestNumber;      // vertical position of number of multimeasure rest
       bool _gap      { false };     // invisible and not selectable for user
       std::vector<NoteDot*> _dots;
 
       virtual QRectF drag(EditData&) override;
       virtual qreal upPos()   const override;
       virtual qreal downPos() const override;
       virtual void setOffset(const QPointF& o) override;
+      virtual Sid getPropertyStyle(Pid pid) const override; 
```

Comment:
no need to be virtual

Reply:
why not keep virtual functions virtual?

Reply:
because the method is already virtual in the base class
it's enough to use override keyword in the inherited classes

Reply:
Then all those above would need to be changed too, and a lot of others in the codebase.
Plus, the `virtual`, being the very first word makes it extra clear. More than the easily missed `override` at the end.

Reply:
I see the same is true of other functions I haven't touched, like downPos(), etc. Should I remove the `virtual` keyword wherever it is unnecessary throughout this whole class, while I'm passing through?

Reply:
@vpereverzev I've added a separate commit removing all unnecessary occurrences of `virtual` in this class.